CMSgov/design-system
# [NO_TICKET] Error placement feature flags

## Summary
- Adds React feature flag functionality
- Renames `FieldContainer` to `FormControl`, reorganizes `InlineError`
- Reorganizes exports
- Remove unnecessary `aria-describedby` in `TextField` and `Dropdown`
- Exports `maskValue` function

## How to test
- Test the doc site to ensure no changes in form component functionality and ensure feature flags are being used in react examples.
http://design-system-demo.s3-website-us-east-1.amazonaws.com/feature/error-placement
- Test the react feature flag in a CRA example and a product app
- Test the react feature flag in a child design system (i.e. the `bernard/feature-flag-test` branch in HCgov DS)
